TgeeT/StackOverflow-lite
# #162562056 user login and signup

#### What does this PR do?
This PR has a login and signup form which need to be integrated to the main project

 #### Description of Task to be completed?
Create a login form
Create a signup form for new users
Create a link for new users in the login page to register

 #### How should this be manually tested?
cloning or forking this repo
check out to ft-user-login-162562056 
Open index.html in a browser

 #### What are the relevant pivotal tracker stories?
#162562056

 #### Screenshots (if appropriate)
![login](https://user-images.githubusercontent.com/45405036/49930049-e5fec100-fed4-11e8-9f49-157aebb41477.png)
![signup](https://user-images.githubusercontent.com/45405036/49930093-f0b95600-fed4-11e8-90a0-cebc24606049.png)
